src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,100 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_CAPACITY = 10;
+    private Object[] innerArray = new Object[DEFAULT_CAPACITY];
```

Comment:
All non-constant fields should be initialized in the constructor
Also, make it final